odl/operator/default_ops.py

```diff
@@ -866,6 +866,321 @@ def __str__(self):
         return '0'
 
 
+class RealPart(Operator):
+
+    """Operator that extracts real part of a vector."""
+
+    def __init__(self, space):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        space : `FnBase`
+            Space which real part should be taken, needs to implement
+            ``space.real_space``.
+
+        Examples
+        --------
+        Take real part of cn vector:
+
+        >>> c3 = odl.cn(3)
+        >>> op = RealPart(c3)
+        >>> op([1 + 2j, 2, 3 - 1j])
+        rn(3).element([1.0, 2.0, 3.0])
+
+        The operator is the identity on real spaces
+
+        >>> r3 = odl.rn(3)
+        >>> op = RealPart(r3)
+        >>> op([1, 2, 3])
+        rn(3).element([1.0, 2.0, 3.0])
+
+        The operator also works on other `FnBase` spaces such as
+        `DiscreteLp` spaces
+
+        >>> r3 = odl.uniform_discr(0, 1, 3, dtype=complex)
+        >>> op = RealPart(r3)
+        >>> op([1, 2, 3])
+        uniform_discr(0.0, 1.0, 3).element([1.0, 2.0, 3.0])
+        """
+        real_space = space.real_space
+        linear = space == real_space
```

Comment:
Maybe it's just me but I find it more readable with the boolean expression in parens.